### ISSUE TITLE
Re-Trainning Model

### ISSUE THREAD

**Issue by sthl** [YadiraF/PRNet#136]
Hello,
First, Thanks for sharing this project code.
Before implementing my model, I am training your code now.
I have already generate the uv map data from 300W-LP(122450 samples)
and modify [.#L39](https://github.com/jnulzl/PRNet/blob/bbdc3c95266aa4ce6379cd343121616244964872/train.py#L39) line like [this](https://github.com/YadiraF/PRNet/pull/114)
Except that I did not change the other parts (learning_rate, epochs, batch_size).
But the results are still do not look like the original version yet.

I cannot sure this is because of 'wrong preprocessing the uv map', 'little training steps' or 'wrong modification on the code'.
How did you set your training code? Could I have your advices?

Upload the final model test image and the training loss from train.py
![screenshot after full dataset](https://user-images.githubusercontent.com/15084660/61997274-2c0e3b00-b0da-11e9-8ad1-c0fddc94b8dc.png)
![training_loss](https://user-images.githubusercontent.com/15084660/61997382-d3d83880-b0db-11e9-9cb3-83a90b5788fd.png)
